code/datums/log_viewer.dm

```diff
@@ -103,17 +111,18 @@
 	var/trStyleTop		= "border-top:2px solid; border-bottom:2px solid; padding-top: 5px; padding-bottom: 5px;"
 	var/trStyle			= "border-top:1px solid; border-bottom:1px solid; padding-top: 5px; padding-bottom: 5px;"
 	var/dat
-	dat += "<head><style>.adminticket{border:2px solid} td{border:1px solid grey;} th{border:1px solid grey;} span{float:left;width:150px;}</style></head>"
-	dat += "<div style='height:15vh'>"
+	dat += "<head><meta http-equiv='X-UA-Compatible' content='IE=edge'><style>.adminticket{border:2px solid} td{border:1px solid grey;} th{border:1px solid grey;} span{float:left;width:150px;}</style></head>"
```

Comment:
Required to use the calc() method. NanoUI also uses this header.
Byond itself is still compatible with windows XP and this is one of the reasons why CSS3 is not supported by default. This fixes that